sdcm/cluster.py

```diff
@@ -4561,6 +4561,26 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
                     if result.ok:
                         self.log.info("Scylla_io_setup result: %s", result.stdout)
 
+                if self.params.get('gce_setup_hybrid_raid'):
+                    gce_n_local_ssd_disk_db = self.params.get('gce_n_local_ssd_disk_db')
+                    gce_pd_ssd_disk_size_db = self.params.get('gce_pd_ssd_disk_size_db')
+                    if not (gce_n_local_ssd_disk_db > 0 and gce_pd_ssd_disk_size_db > 0):
+                        msg = f"Hybrid RAID cannot be configured without NVMe ({gce_n_local_ssd_disk_db}) " \
+                              f"and PD-SSD ({gce_pd_ssd_disk_size_db})"
+                        raise ValueError(msg)
+
+                    # The script to configure a hybrid RAID is named "hybrid_raid.py" and located on the private repository.
+                    hybrid_raid_script = "hybrid_raid.py"
+                    target_path = os.path.join("/tmp", hybrid_raid_script)
+                    node.remoter.send_files(src=f"scylla-qa-internal/custom_d1/{hybrid_raid_script}", dst=target_path)
```

Comment:
something like that would be better
```suggestion
                    node.remoter.send_files(src=f"{get_sct_root_path()}/scylla-qa-internal/custom_d1/{hybrid_raid_script}", dst=target_path)
```

we can't really control in which directory we are, some code might change it without us knowing about it.

Reply:
@fruch , fixed.